rsc/src/main/scala/rsc/report/Messages.scala

```diff
@@ -345,3 +345,13 @@ final case class DefnMethodNotype(outline: DefnMethod, warn: Boolean = true) ext
   def pos = outline.point
   def text = s"No type found at $pos for definition: $outline"
 }
+
+final case class DefnClassInitNotype(
```

Comment:
How about calling this `DefnClassParentInitTypeParametersMissing` or `DefnClassParentTypeParametersMissing`, since the `NoType` type isn't actually involved.